react-effects/my-app/src/List.tsx

```diff
@@ -17,15 +17,16 @@ export function List() {
       try {
         const data = await readItems();
         setItems(data);
-        setError(null);
+        setIsLoading(false);
```

Comment:
Since this is called in the finally block you don't need it here.

Reply:
ohhhh okay! fixed!